src/lib/autoinstall/clients/inst_autosetup_upgrade.rb

```diff
@@ -226,6 +226,23 @@ def main
           false
         )
 
+        #
+        # Checking if at least one base product has been selected/evaluated.
+        #
+        begin
+          Product.FindBaseProducts
+        rescue StandardError
+          msg = _("No new base product has been set.\n" \
```

Comment:
Could we somehow reuse (or even better: share) the same message from `inst_autoinit.rb` above?

Reply:
We have a also AY updates without any AY configuration file. For that the text does not really fit. What do you think ?

Reply:
So maybe use a bit different message?